crates/driver/src/lib.rs

```diff
@@ -0,0 +1,2 @@
+pub mod api;
+pub mod arguments;
```

Comment:
I guess we are planning on using this as a library so it can be controlled in e2e tests? If that isn't the case then we don't need the library and will have save some time compiling.

Reply:
@nlordell and I thought about how the driver could be used by 3rd parties in the future.
1. as a separate process running an HTTP server (in container or so)
2. as a dependency in a rust project
3. via FFI in any kind of project

2 and 3 would require the lib. Although it's not required immediately I think it would make sense to already have the lib.
The scenario with the e2e tests didn't cross my mind, yet. But that also a good reason. 👍